src/result.rs

```diff
@@ -81,6 +89,35 @@ impl QueryResultWrapper {
       .map(|f| f.name.clone())
       .collect()
   }
+
+  #[napi]
+  pub fn get_columns_specs(&self) -> Vec<MetaColumnsWrapper> {
+    self
+      .internal
+      .col_specs()
+      .iter()
+      .map(|f| {
+        MetaColumnsWrapper {
+          // ToDo: Check if rust provides such information
+          ksname: "unknown?".to_string(),
```

Comment:
`f.table_spec.keyspace_name().to_owned()`

Reply:
I'm against using `to_string()` for `&str` -> `String` conversions. `to_string()` is too powerful and it suggests some stringification, such as `u32` -> `String`.